### PR TITLE
Project modules

### PR DESCRIPTION
**Pull request by pDiller**
Basic Wicket setup.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -8,9 +8,14 @@ repositories {
...
+
     compile 'com.google.guava:guava:20.0'
     compile 'com.atlassian.jira:jira-rest-java-client:2.0.0-m2'
-    testCompile 'junit:junit:4.12'
-    testCompile group: 'org.assertj', name: 'assertj-core', version: '3.8.0'
-}
+    compile 'org.apache.wicket:wicket:8.0.0-M6'
+    compile 'javax.servlet:javax.servlet-api:3.1.0'
 
+    testCompile 'junit:junit:4.12'// https://mvnrepository.com/artifact/org.mockito/mockito-all
```
Comment (matthiasbalke) [inline build.gradle]: What is this comment for? Should it be removed?

File: build.gradle [hunk 2]
```diff
@@ -8,9 +8,14 @@ repositories {
...
+    testCompile 'org.mockito:mockito-all:1.9.5'
+    testCompile 'org.assertj:assertj-core:3.8.0'
+}
```
Comment (matthiasbalke) [inline build.gradle]: You could think about always ending files with newlines. (If the newline is missing, github displays this little red icon). This should be easy to automate by a code formatter, so you don't have to care about it.

File: src/main/java/de/reflectoring/jiraalerts/application/JiraAlertsApplication.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package de.reflectoring.jiraalerts.application;
+
+import de.reflectoring.jiraalerts.base.JiraAlertsHomePage;
+import org.apache.wicket.Page;
+import org.apache.wicket.protocol.http.WebApplication;
+
+public class JiraAlertsApplication extends WebApplication{
```
Comment (matthiasbalke) [inline src/main/java/de/reflectoring/jiraalerts/application/JiraAlertsApplication.java]: you should define a simple code formatter right from the beginning :) how about google code formatter?

File: .gitignore [hunk 4]
```diff
@@ -30,3 +30,6 @@ build
 *.ipr
 *.iws
 *.iml
+
+#output directory
+out
```
Comment (matthiasbalke) [inline .gitignore]: do you really need the excludes at the beginning? If you never package e.g. an ear, you could omit these entries for clarity.

File: build.gradle (hunk 1 above)
Comment (pDiller) [inline build.gradle]: Done it. Should be in with next commit.

File: .gitignore (hunk 4 above)
Comment (pDiller) [inline .gitignore]: you´re right. I think i will remove them with ne next commit. thank you!

File: src/main/java/de/reflectoring/jiraalerts/application/JiraAlertsApplication.java (hunk 3 above)
Comment (pDiller) [inline src/main/java/de/reflectoring/jiraalerts/application/JiraAlertsApplication.java]: hmpf... the code is not formatted, ´cause i always commit with console. Thank you for this!

**Comment by pDiller** [comment]
@matthiasbalke @danielvdwal i´ve solved the comments from Matthias. Please make a review so i can merge the pullrequest.

**Pull request merged**